src/seedu/addressbook/ui/TextUi.java

```diff
@@ -21,18 +20,14 @@
     /** A decorative prefix added to the beginning of lines printed by AddressBook */
     private static final String LINE_PREFIX = "|| ";
 
+    /** Offset required to convert between 1-indexing and 0-indexing.  */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+    
     /** A platform independent line separator. */
     private static final String LS = System.lineSeparator();
 
     private static final String DIVIDER = "===================================================";
```

Comment:
May be even things like this should be moved to the `Formatter`? Note that it may not be clear if something belongs in the `Formatter` class or the `TextUi` class. It depends on how you define the responsibility of the `Formatter` class.